data/communities.json

```diff
@@ -136,5 +133,110 @@
   "codeOfConduct": false,
   "levels": ["beginner"],
   "tags": ["Code", "Refugees", "Inclusion", "Diversity"]
-  }
+  },
+  {
+  "name": "Django Girls",
+  "description": "We inspire women to fall in love with programming.  Django Girls empowers and helps women to organize free, one-day programming workshops.",
+  "url": "https://djangogirls.org",
+  "logo": "https://djangogirls.org/static/img/global/logo.png",
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/djangogirls"
+    }
+  ],
+  "codeOfConduct": "https://djangogirls.org/coc/",
+  "levels": ["beginner"],
+  "tags": ["Code", "Women", "Inclusion", "Diversity"]
+  }, 
+  {
+  "name": "Pyladies",
+  "description": "We are an international mentorship group with a focus on helping more women become active participants and leaders in the Python open-source community.",
+  "url": "http://www.pyladies.com",
+  "logo": "http://www.pyladies.com/assets/images/pyladies_logo.png",
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/PyLadiesLondon"
+    }
+  ],
+  "codeOfConduct": "http://www.pyladies.com/CodeOfConduct/",
```

Comment:
I had planned for `codeOfConduct` to be a `boolean` but I prefer your idea of putting a link 👍 . I will update the ones that don't have a CodeOfConduct to an empty string.